src/lib/state/swap.ts

```diff
@@ -27,3 +28,6 @@ export const independentFieldAtom = pickAtom(swapAtom, 'independentField')
 
 // If set to a transaction hash, that transaction will display in a status dialog.
 export const displayTxHashAtom = atom<string | undefined>(undefined)
+
+export const DEFAULT_FEE_OPTIONS = undefined
```

Comment:
nit: Considering that this is undefined, I'd be fine if you just used the value `undefined` inline throughout, instead of exporting it.

Reply:
👍